src/components/SideNavBar/SideNavBarTop.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import cn from 'classnames';
+import Icon from '../Icon/Icon';
+import locationPin from '../Icon/icons/locationPin';
+import style from './style.scss';
+
+const SideBarNavTop = ({ label, onClick }) => {
+  return (
+    <div
+      className={style.top}
+      onClick={onClick}
+    >
+      <div className={style.iconAndLabelContainer}>
+        <div className={style.iconWrapper}>
+          <Icon
+            color="#fff"
+            icon={locationPin}
```

Comment:
seem to be mixing spacestation-specific stuff in plasma - shouldn't ssv2 be passing in `locationPin` to the component?